pkg/sdk/security_integrations_def.go

```diff
@@ -307,15 +308,14 @@ var scimIntegrationSetDef = g.NewQueryStruct("ScimIntegrationSet").
 	OptionalBooleanAssignment("ENABLED", g.ParameterOptions()).
 	OptionalIdentifier("NetworkPolicy", g.KindOfT[AccountObjectIdentifier](), g.IdentifierOptions().Equals().SQL("NETWORK_POLICY")).
 	OptionalBooleanAssignment("SYNC_PASSWORD", g.ParameterOptions()).
-	OptionalComment().
+	OptionalQueryStructField("Comment", stringAllowEmptyDef, g.ParameterOptions().SQL("COMMENT")).
```

Comment:
Not sure about this change, what is it for?

Reply:
Ah, I remember, set comment wasn't produced for empty comment, right? In that case you can just compare comment field contents like so `d.Get("comment").(string) == "")` and if it's true you can call unset otherwise set.

Reply:
As I wrote above, UNSET COMMENT is not working for this resource, that's why we need to support SET COMMENT=''. Without this change this SQL is not generated.